firmware/Makefile

```diff
@@ -10,6 +11,20 @@ build: FORCE
 clean: FORCE
 	rm $(BUILD_DIR) -rf
 
+deepclean: clean
+ifeq ($(findstring stm32,$(PLATFORM)),stm32)
+	@echo "Deleting files and directories in $(CUBEMX_DIR) that are ignored by Git"
+	@cd $(CUBEMX_DIR) && \
```

Comment:
we do not need the command to be a one-liner, please split into separate commands (remove the `&& \`)